schemas/ingest-replay-events.v1.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$ref": "#/definitions/ReplayEvent",
+  "title": "replay_event",
+  "definitions": {
+    "ReplayEvent": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["something", "replay_event"]
```

Comment:
This looks kinda wrong. are we really sending type=something in prod?

Reply:
oh shit, oops

Reply:
i hand wrote that

Reply:
Updated. I only actually saw `replay_event` in prod